app/views/media/_manuscript_card.html.erb

```diff
@@ -62,6 +62,13 @@
             </span>
           </div>
         </div>
+        <% if medium.sort == 'Nuesse' && medium.file_last_edited != nil %>
```

Comment:
I would suggest putting this in a card-footer, see [here](https://getbootstrap.com/docs/4.6/components/card/#header-and-footer).